docs/conf.py

```diff
@@ -0,0 +1,70 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('../'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Linux Integration Services Automation (LISA)'
+copyright = '2021, Microsoft'
+author = 'Microsoft'
+
+release = ''
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinxemoji.sphinxemoji',
+    'sphinx.ext.autosectionlabel',
+]
```

Comment:
I suggest "sphinx_copybutton", to begin with (I have others in my tree as well, but I'm also learning :))

Reply:
Thanks, Gustavo. I will take a look at that.

Reply:
Added. Thanks again :-)